docs/UserGuide.md

```diff
@@ -392,29 +481,24 @@ Format: `alias -c <existing_command> -as <alias>`
 * `<existing_command>` refers to any command keywords e.g. `search`, `addalloc`, `addstudent`.
 * The `<alias>` can only be a single alphanumeric word.
 * Multiple aliases can be set for each command.
-* Default and existing aliases can still be used after new alias is added. e.g. after `alias -c addstudent -a as`, both `as` and `addstudent` can be used.
+* Default and existing aliases can still be used after new alias is added. E.g. after `alias -c addstudent -as student`, both `student` and `addstudent` can be used.
 * The `<existing_command>` can also take any existing alias as an input. The new alias will be added as an alternative to the default command.
-* The aliases created will persist between Source Control sessions.
-* If you find that you have too many aliases, you can remove an alias by using `alias -c <alias> -as <same_alias>`
+* If you find that you have too many aliases, you can remove an alias by using `alias -c <alias> -as <same_alias>`.
+* The aliases created will persist between Source Control sessions, as long as the application is in the same directory as it was previously. 
 
 Examples:
-* `alias -c addstudent -as as` adds a new alias to `addstudent` command.
-  * `as -n Zhiying -i E1234567` adds student `Zhiying` to database.
-* `alias -c addscore -as as` replace the mapping of the alias `as` to the `addscore` command. i.e. `as` no longer function as `addstudent`.
-  * `as -a P01 -n Zhiying -s 80` will add a score for `P01` for the student `Zhiying`.
-* `alias -c as -as as2` will add the alias `as2` to the command that `as` currently maps to. i.e. `as2` functions as `addscore`.
-* `alias -c as -as as` will remove the alias `as`. `as` will no longer be recognised as a command.
-
-[Return to table of contents](#table-of-contents)
-
-
-### Saving the data
-
-Source Control data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+* `alias -c addstudent -as example` 
+  * adds a new alias to `addstudent` command.
+  * `example -n Zhiying -i E1234567` adds student `Zhiying` to database.
```

Comment:
Just one v v small thing 🙏🏻 other than this I think it is okay to merge! So I think here it should be "will add student..." instead to match the "pattern" of the other two examples below.

```suggestion
  * `example -n Zhiying -i E1234567` will add student `Zhiying` to the database.
```